.rubocop_todo.yml

```diff
@@ -1,19 +1,12 @@
 # This configuration was generated by
```

Comment:
can we not add new todos ?

Reply:
I was a bit torn about it.
When I got [the Rubocop warnings](https://github.com/zendesk/predictive_load/actions/runs/6111925065/job/16588243736) I started addressing them, but then I realized that no other file in this codebase is really paying attention to Rubocop.

For example, it complained about missing top-level documentation comments for modules, but no other module in the repo has them. Or it complained about double-quoted strings without interpolation, but I used them only to keep things consistent within the files, and in fact the codebase uses both double and single quoted strings throughout.

So yes, I could address the warnings introduced in the PR diff, but I wonder if it would leave the codebase in a more inconsistent state.

Reply:
Waiting for https://github.com/zendesk/predictive_load/pull/33 to be merged. I'll rebase then, and address any violation.